NEWS

```diff
@@ -1,6 +1,10 @@
 PHP                                                                        NEWS
 |||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||
-?? ??? ????, PHP 7.1.19
+?? ??? ????, PHP 7.1.20
+
+
+
+22 Jun 2019, PHP 7.1.19
```

Comment:
This should probably be `22 Jun 2018`

Reply:
Like @cmb69 said, the PHP version is updated by the RMs and NEWS are updated by the PR merger to not cause conflicts. This PR should be closed in favor of #3277